tasks/managed/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml

```diff
@@ -180,33 +180,112 @@ spec:
             fi
         }
 
+        # Function to download SBOM for a single image
+        download_sbom_for_image() {
+            local pyxis_image="$1"
+            local image_url="$2"
+            
+            local file
+            local digest
+            local arch_digest
+            local os
+            local arch
+            local platform
+            
+            file="$(jq -r '.imageId' <<< "$pyxis_image").json"
+            digest="$(jq -r '.digest' <<< "$pyxis_image")"
+            arch_digest="$(jq -r '.arch_digest' <<< "$pyxis_image")"
+            
+            # cosign has very limited support for selecting the right auth entry,
+            # so create a custom auth file with just one entry.
+            DOCKER_CONFIG="$(mktemp -d)"
+            export DOCKER_CONFIG
```

Comment:
Since this is now done each time in the function, you should probably remove the same thing that one on line 159-160.